src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,10 @@
+const Footer = () => {
+    const year = new Date().getFullYear()
+    return <footer className="flex justify-center items-center pb-[22px]" >
+        <div className="p-3 border-y-[1px] border-primary">
+            <p className="text-bold text-[20px] font-nav">{year} Galois Derivaties | Email: hello@galois.cloud</p>
```

Comment:
please have Email as a mailto link

Reply:
Done. Also added animation to it so that when it hovers, it changes to 'contact us', which is also a mailto link